Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -27,46 +23,15 @@ finitely many vertices.
 
 * `SimpleGraph.commonNeighbors` is the intersection of the neighbor sets of two given vertices
 
-* `SimpleGraph.neighborFinset` is the `Finset` of vertices adjacent to a given vertex,
-   if `neighborSet` is finite
-
 * `SimpleGraph.incidenceSet` is the `Set` of edges containing a given vertex
 
-* `SimpleGraph.incidenceFinset` is the `Finset` of edges containing a given vertex,
-   if `incidenceSet` is finite
-
 * `SimpleGraph.Dart` is an ordered pair of adjacent vertices, thought of as being an
   orientated edge. These are also known as "half-edges" or "bonds."
 
-* `SimpleGraph.Hom`, `SimpleGraph.Embedding`, and `SimpleGraph.Iso` for graph
-  homomorphisms, graph embeddings, and
-  graph isomorphisms. Note that a graph embedding is a stronger notion than an
-  injective graph homomorphism, since its image is an induced subgraph.
-
 * `CompleteAtomicBooleanAlgebra` instance: Under the subgraph relation, `SimpleGraph` forms a
   `CompleteAtomicBooleanAlgebra`. In other words, this is the complete lattice of spanning subgraphs
   of the complete graph.
 
-## Notations
-
-* `→g`, `↪g`, and `≃g` for graph homomorphisms, graph embeddings, and graph isomorphisms,
-  respectively.
-
-## Implementation notes
-
-* A locally finite graph is one with instances `Π v, Fintype (G.neighborSet v)`.
-
-* Given instances `DecidableRel G.Adj` and `Fintype V`, then the graph
-  is locally finite, too.
-
-* Morphisms of graphs are abbreviations for `RelHom`, `RelEmbedding`, and `RelIso`.
-  To make use of pre-existing simp lemmas, definitions involving morphisms are
-  abbreviations as well.
-
-## Naming Conventions
-
-* If the vertex type of a graph is finite, we refer to its cardinality as `CardVerts`.
```

Comment:
Please make sure this makes it into Finite. Also please modify it to say "as `CardVerts` or `card_verts`".